core/orchestrator.go

```diff
@@ -522,6 +551,7 @@ func (rtm *RemoteTranscoderManager) Register(transcoder *RemoteTranscoder) {
 
 func (rtm *RemoteTranscoderManager) Unregister(t *RemoteTranscoder) {
 	rtm.RTmutex.Lock()
+	// Probably should remove from remoteTranscoders too?
```

Comment:
We fix up the list during selection time to avoid iterating under the lock. (Also, since the multi-T PR is still in review, welcome to submit comments there)